### PR TITLE
feat: 🎸 make 22-patch a prerelease

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

Attempt to get around this restriction: https://github.com/PolymeshAssociation/polymesh-sdk/issues/1103#issue-2032715327

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: release.config.js [hunk 1]
```diff
@@ -2,7 +2,10 @@ module.exports = {
   repositoryUrl: 'https://github.com/PolymeshAssociation/polymesh-sdk.git',
   branches: [
     'master',
-    '22-patch',
+    {
+      name: '22-patch',
+      prerelease: true,
```
[the remark this answers is not in the document]
Comment (prashantasdeveloper) [reply release.config.js:7]: https://github.com/PolymeshAssociation/polymesh-sdk/blob/lts/release.config.js#L15

[the remark this answers is not in the document]
Comment (prashantasdeveloper) [reply release.config.js:7]: I faced the same issue when I was adding support for lts

**Review by prashantasdeveloper — CHANGES_REQUESTED** [review]

File: release.config.js [hunk 2]
```diff
@@ -2,7 +2,11 @@ module.exports = {
   repositoryUrl: 'https://github.com/PolymeshAssociation/polymesh-sdk.git',
   branches: [
     'master',
-    '22-patch',
+    {
+      name: 'patch',
+      prerelease: true,
```
Comment (polymath-eric) [inline release.config.js:7]: I don't think we need this as pre-release if the range works like I think it does

**Pull request merged**